sdk/tests/conformance2/textures/misc/copy-texture-image.html

```diff
@@ -0,0 +1,174 @@
+        "R8I",
+        "R16I",
+        "R32I",
+        "RG8I",
+        "RG16I",
+        "RG32I",
+        "RGBA8I",
+        "RGBA16I",
+        "RGBA32I",
+    ];
+    var goodUnsingedIntFormats = [
+        "R8UI",
+        "R16UI",
+        "R32UI",
+        "RG8UI",
+        "RG16UI",
+        "RG32UI",
+        "RGB10_A2UI",
+        "RGBA8UI",
+        "RGBA16UI",
+        "RGBA32UI",
+    ];
+    var badFormats = [
+        "R8_SNORM",
+        "R16F",
+        "R32F",
+        "RG8_SNORM",
+        "RG16F",
+        "RG32F",
+        "RGB8_SNORM",
+        "R11F_G11F_B10F",
+        "RGB9_E5",
+        "RGB16F",
+        "RGB32F",
+        "RGB8UI",
+        "RGB8I",
+        "RGB16UI",
+        "RGB16I",
+        "RGB32UI",
+        "RGB32I",
+        "RGBA8_SNORM",
+        "RGBA16F",
+        "RGBA32F",
+        "DEPTH_COMPONENT16",
+        "DEPTH_COMPONENT24",
+        "DEPTH_COMPONENT32F",
+        "DEPTH24_STENCIL8",
+    ];
+
+    var texture = gl.createTexture();
+    gl.bindTexture(gl.TEXTURE_2D, texture);
+
+    function testFormat (internalformat, srcTexFormatsTypes) {
+        var fbo = gl.createFramebuffer();
+        var srcTexture = gl.createTexture();
+        gl.bindTexture(gl.TEXTURE_2D, srcTexture);
+        gl.texImage2D(gl.TEXTURE_2D, 0, srcTexFormatsTypes.internalformat, 64, 64, 0, srcTexFormatsTypes.format, srcTexFormatsTypes.type, null);
+        gl.bindFramebuffer(gl.FRAMEBUFFER, fbo);
+        gl.framebufferTexture2D(gl.FRAMEBUFFER, gl.COLOR_ATTACHMENT0, gl.TEXTURE_2D, srcTexture, 0);
+
+        gl.bindTexture(gl.TEXTURE_2D, texture);
+        gl.copyTexImage2D(gl.TEXTURE_2D, 0, gl[internalformat], 0, 0, 64, 64, 0);
+        wtu.glErrorShouldBe(gl, gl.NO_ERROR, "copyTexImage2D should succeed for good internalformat " + enumToString(gl[internalformat]));
+
+        gl.deleteTexture(srcTexture);
+        gl.deleteFramebuffer(fbo);
+    }
+
+    goodByteFormats.forEach(function(internalformat) {
+        var srcTexFormatsTypes = { internalformat: gl.RGBA, format: gl.RGBA, type: gl.UNSIGNED_BYTE };
+        testFormat(internalformat, srcTexFormatsTypes);
+    });
+    goodSRGBFormats.forEach(function(internalformat) {
+        var srcTexFormatsTypes = { internalformat: gl.SRGB8_ALPHA8, format: gl.RGBA, type: gl.UNSIGNED_BYTE };
+        testFormat(internalformat, srcTexFormatsTypes);
+    });
+    goodIntFormats.forEach(function(internalformat) {
+        var srcTexFormatsTypes = { internalformat: gl.RGBA32I, format: gl.RGBA_INTEGER, type: gl.INT };
+        testFormat(internalformat, srcTexFormatsTypes);
+    });
+    goodUnsingedIntFormats.forEach(function(internalformat) {
+        var srcTexFormatsTypes = { internalformat: gl.RGBA32UI, format: gl.RGBA_INTEGER, type: gl.UNSIGNED_INT };
+        testFormat(internalformat, srcTexFormatsTypes);
+    });
+    badFormats.forEach(function(internalformat) {
+        gl.copyTexImage2D(gl.TEXTURE_2D, 0, gl[internalformat], 0, 0, 64, 64, 0);
```

Comment:
I think you should set up just like the good internalformat test for the bad ones.  Otherwise the error may be generated from somewhere else and you are not actually testing what you intend to test.